src/Events/Controllers/AppController.cs

```diff
@@ -68,8 +68,9 @@ public class AppController : ControllerBase
         [Produces("application/json")]
         public async Task<ActionResult<string>> Post([FromBody] CloudEventRequestModel cloudEvent)
         {
-            if (string.IsNullOrEmpty(cloudEvent.Source.OriginalString) || string.IsNullOrEmpty(cloudEvent.SpecVersion) ||
-            string.IsNullOrEmpty(cloudEvent.Type) || string.IsNullOrEmpty(cloudEvent.Subject))
+            if (string.IsNullOrEmpty(cloudEvent.Source?.OriginalString) ||
+                string.IsNullOrEmpty(cloudEvent.SpecVersion) ||
+                string.IsNullOrEmpty(cloudEvent.Type) || string.IsNullOrEmpty(cloudEvent.Subject))
```

Comment:
would suggest each condition on a single line if we are to seperate them on individual lines

Reply:
Agreed.